### PR TITLE
Modal to lock comments on project|package|request

### PR DESCRIPTION
**Pull request by ncounter**
lock-unlock

### CODE REVIEW

**Review by krauselukas — DISMISSED** [review]
I get an error when I comment on a request or project that has a comment lock as a Admin user

**Comment by krauselukas** [comment]
> I get an error when I comment on a request or project that has a comment lock as a Admin user

on a request i get the following: 
```
ActionView::Template::Error (undefined method `utc' for nil:NilClass):
1: = render(BsRequestCommentComponent.new(comment: comment.root, level: 1, commentable: commentable))
```
on a project a code `422`

**Comment by ncounter** [comment]
@krauselukas @hellcp-work rebase is done, I also updated the description with a short recording of the behavior

File: src/api/app/views/webui/comment/_comment_field.html.haml
Comment (ncounter) [reply src/api/app/views/webui/comment/_comment_field.html.haml:4]: It's a leftover of the process during implementation of the final solution. I'm getting rid of it now

File: src/api/app/views/webui/comment/_comment_field.html.haml
Comment (ncounter) [reply src/api/app/views/webui/comment/_comment_field.html.haml:2]: This cannot be done because `modal_id` is used in two places, on very deep inside the `form` where the `Lock comment` button lies in, and one at the bottom but outside of the `form`, as a parameter of the partial for the modal.

**Review by krauselukas — COMMENTED** [review]
@ncounter I merged the PR with the controller part, after rebasing this PR, I think its good to go

**Comment by ncounter** [comment]
> @ncounter I merged the PR with the controller part, after rebasing this PR, I think its good to go

@krauselukas great, thanks :+1: . I just rebased and pushed, let's wait for the checks now and an approval before merging this too.

**Review by danidoni — APPROVED** [review]

**Pull request merged**
